src/main/java/thePackmaster/powers/highenergypack/InvestorPower.java

```diff
@@ -0,0 +1,36 @@
+package thePackmaster.powers.highenergypack;
+
+
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class InvestorPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("InvestorPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public InvestorPower(int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, AbstractDungeon.player, amount);
+    }
+
+    @Override
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        if (card.costForTurn >= 2 && !card.freeToPlayOnce) {
```

Comment:
```suggestion
        if (Wiz.getLogicalCardCost(card) >= 2) {
```
Little helper method that has some more logic for this